generate_and_apply_patch.sh

```diff
@@ -6,6 +6,23 @@ BOOT=${1:-/boot}
 IRFS_HOOK="/etc/initramfs-tools/hooks/acpi_override.sh"
 cd `mktemp -d`
 
+if [ "$(iasl -v | grep -c 20180811)" -eq 0 ]; then
+ #Build a newer version of iasl
+ if [ -x $(which apt-get) ]; then
+     echo "[*] Installing required tools to build iasl"
+     sudo apt-get -y install bison flex
+ fi
+ echo "[*] Upgrading IASL..."
+ wget https://acpica.org/sites/acpica/files/acpica-unix2-20180810.tar.gz
+ tar -zxf acpica-unix2-20180810.tar.gz
+ cd acpica-unix2-20180810
+ make clean
+ make iasl || { echo 'ERROR: Failed to build IASL' ; exit 1; }
+ sudo make install || { echo 'ERROR: Failed to install new version of IASL' ; exit 1; }
```

Comment:
it would be great if we didn't force the install globally here, I for one generally never want manually compiled things to be installed globally like that, I'd much rather install locally in a specific area when I do it myself, if I'm not doing it myself I want the newly compiled thing to be 'installed' local to the thing I'm working with...

Reply:
Fair point - Changing